nixos/modules/security/pam.nix

```diff
@@ -397,6 +434,10 @@ let
                  " kwalletd=${pkgs.kdeFrameworks.kwallet.bin}/bin/kwalletd5")}
               ${optionalString cfg.enableGnomeKeyring
                 "auth optional ${pkgs.gnome3.gnome-keyring}/lib/security/pam_gnome_keyring.so"}
+              ${optionalString cfg.gnupg.enable
+                "auth optional ${pkgs.pam_gnupg}/lib/security/pam_gnupg.so"
+                + optionalString (cfg.gnupg.enable && cfg.gnupg.storeOnly) " store-only"
```

Comment:
What about `cfg.gnupg.noAutostart`, shouldn't it be here as well?

Reply:
Nope, `no-autostart` is only for the lines beginning with `session`; similarly, `store-only` is only for lines beginning with `auth`.